packages/TelegramClientTests-Core.package/TCTCMessageTests.class/instance/testPhotoMessageCorrectlyConstructedFromJson.st

```diff
@@ -0,0 +1,14 @@
+testing
+testPhotoMessageCorrectlyConstructedFromJson
+
+	| photoMessageEvent chat photoMessage |
+
+ 	chat := TCTMMocks mockPrivateChat.
+	photoMessageEvent := TCTMMocks mockPhotoMessageJsonFrom: chat id.
+	photoMessage := TCCMessage newFromMessageEvent: photoMessageEvent in: chat with: self core.
+
+	self assert: TCTMMocks mockText equals: photoMessage caption.
+	self assert: photoMessage hasCaption.
+	self assert: 3 equals: photoMessage fileId.
+	self assert: 200 equals: photoMessage height.
+	self assert: 400 equals: photoMessage width.
```

Comment:
Where do 3, 200 and 400 come from? They make it hard for me to understand the test.

Reply:
fixed